canary/src/main/java/io/littlehorse/canary/config/CanaryConfig.java

```diff
@@ -0,0 +1,48 @@
+package io.littlehorse.canary.config;
+
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+public class CanaryConfig implements Config {
+
+    public static final String LH_CANARY_PREFIX = "lh.canary.";
```

Comment:
Ah, I see, `LHC_` conflicts with LH Client configurations. That is interesting. However, I don't know if I've ever seen environment variables with `.` in them...

I'm not sure how critical it is for the canary to support environment variable configs. The Server does, and we use that in the operator. But our operator could easily make a Properties file for the canary to run as well. What do you think?